meeting-notes/2019/Q1/2019-01-21--js-core-dev-team-weekly.md

```diff
@@ -0,0 +1,181 @@
+# ⚡️ⒿⓈ Core Dev Team Weekly Sync 🙌🏽 January 21, 2019
+
+- **Lead:** @alanshaw
+- **Notetaker:** @jacobheun
+- **Attendees:**
+  - @jacobheun
+  - @vasco-santos
+  - @vmx
+  - @lidel
+  - @chriscool
+  - @mcollina
+  - @achingbrain
+- **Recording:** https://youtu.be/5Ka4b5WG2BM
+
+## Agenda
+
+- Ask everyone to put their name into the list of attendees
+- Round of updates
+  - What have you accomplished since the last Weekly?
+  - Were there any blockers? If so, which ones? Is it still blocked? Why?
+  - What is the next important thing you should focus on?
+- Ask for general questions. Could be things like:
+  - I'm stuck with something, I don't know who to ask. Who knows who to ask?
+  - Who can help me with xyz?
+- Plan this week
+  - Select issues to work on
+- Review remaining issues if there is time left
+
+## Notes
+
+### Week Update
+
+@alanshaw
+- Done:
+    - Resolved issue with switch to bignumber.js ([#187](https://github.com/ipfs/js-ipfs-repo/pull/187)) to unblock [#1803](https://github.com/ipfs/js-ipfs/pull/1803) for 0.34 release
+    - Opened PR for disabling preload in test/CI environments [#1819](https://github.com/ipfs/js-ipfs/pull/1819)
+    - Fixed `hlsjs-ipfs-loader` for [`browser-video-streaming` example](https://github.com/moshisushi/hlsjs-ipfs-loader/pull/7)
+    - Many fixes for JS IPFS examples
+    - Released js-ipfs 0.34 🚢💨 🎉
+    - Created "lazily loaded IPLD formats from IPFS" proof of concept see https://github.com/alanshaw/ipld-formats and https://youtu.be/ADA_HT2whok
+    - Enabled browser builds for `ipld-ethereum` [#43](https://github.com/ipld/js-ipld-ethereum/pull/43) and `ipld-git` [#39](https://github.com/ipld/js-ipld-git/pull/39)
+    - Fixed illusive "Database is not open" error [#1834](https://github.com/ipfs/js-ipfs/pull/1834)
+    - Added feature `echo "hello" | jsipfs add` [#1833](https://github.com/ipfs/js-ipfs/pull/1833)
+- Blocked:
+    - `null`
+- Next:
+    - Timeline for JS IPFS roadmap
+    - Finally get up to speed with IPFS benchmarks and aim to write a benchmark
+    - Start CID v1 base32 default work ([step 2 here](https://github.com/ipfs/js-ipfs/issues/1440#issuecomment-407343797))
+
+@jacobheun
+- Done:
+	- Libp2p Team Week all last week
+    - Created a rough roadmap for rolling out [Rendezvous](https://gist.github.com/jacobheun/13247de56e61e717c8a6d682406d01bf)
+      - Will work on improving that and will turn that into an Awesome Endeavor issue this week.
+    - We finalized a draft of the [rendezvous spec](https://github.com/libp2p/specs/blob/e1083c1f9d8f7afc0d65a43a12b05492f3873385/rendezvous/README.md), which will enable us to finish up the initial work on the rendezvous go and js implementations
+- Blocked: N/A
+- Next:
+  - Create `Awesome Endeavor` issue to track the release of Rendezvous. 
```

Comment:
Url?

Reply:
A bit behind but I have the initial Roamap up in the libp2p repo https://github.com/libp2p/libp2p/issues/66